tests/lib/processor.js

```diff
@@ -514,13 +514,13 @@ describe("processor", function() {
         ].join("\n");
         var messages = [
             [
-                { line: 1, column: 1, message: "Use the global form of \"use strict\".", ruleId: "strict" },
-                { line: 3, column: 5, message: "Unexpected console statement.", ruleId: "no-console" }
+                { line: 1, endLine: NaN, column: 1, message: "Use the global form of \"use strict\".", ruleId: "strict" },
```

Comment:
Now that #89 dropped support for ESLint < v4, we should be able to test without any `NaN` here, so this would be 1, and the assertion below would become 4.